src/components/datasets-list/datasets-item-info/index.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import { useLayerSource } from '@/hooks/map';
+
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/ui/popover';
+import Icon from 'components/icon';
+import INFO_SVG from 'svgs/ui/info.svg?sprite';
+
+export const DatasetsItemInfo = ({ layer_id }: { layer_id: string }) => {
```

Comment:
```suggestion
export const DatasetsItemInfo : FC<{ layer_id: string }> = ({ layer_id }) => {
```
Here the const is typed, there are more alternatives, but this is the best way to type. In the other way, you are forcing the inferred types.